ocs_ci/helpers/dr_helpers.py

```diff
@@ -1265,10 +1265,34 @@ def get_all_drpolicy():
         list: List of all DRPolicy
 
     """
+    return_drpolicy_list = []
+    current_managed_clusters_list = []
     config.switch_acm_ctx()
+    acm_hub_name = config.current_cluster_name()
     drpolicy_obj = ocp.OCP(kind=constants.DRPOLICY)
     drpolicy_list = drpolicy_obj.get(all_namespaces=True).get("items")
-    return drpolicy_list
+    for cluster_name in config.clusters:
+        if cluster_name.ENV_DATA.get("rbd_dr_scenario"):
+            current_managed_clusters_list.append(
+                cluster_name.ENV_DATA.get("cluster_name")
+            )
+    dr_cluster_relations = config.MULTICLUSTER.get("dr_cluster_relations", [])
+    if dr_cluster_relations:
+        current_managed_clusters_list = [
+            f"{constants.HYPERSHIFT_ADDON_DISCOVERYPREFIX}-{item}"
+            for item in dr_cluster_relations[0]
```

Comment:
dr_cluster_relations[0] is applicable for DR setup stage only, where dr_cluster_relations will have only one element or the first element will be taken for installation. There will be one or more than one pairs(more than one element in  dr_cluster_relations) in a test run. Approving the changes for the time being, please add a TODO to handle more than one element in dr_cluster_relations.

Reply:
I have created a issue for this